### ISSUE TITLE
[BUG] WebGL with running Chrome headless in Docker playwright v.1.16.0

### ISSUE THREAD

**Issue by JoFrMueller** [microsoft/playwright#10084]
**Context:**
- Playwright Version: 1.16.0
- Operating System: Windows + Docker
- Node.js version: The one that comes with the mcr.microsoft.com/playwright:v1.16.0-focal Docker image
- Browser: Chromium

**Code Snippet**

The code snippet doesn't play any role. We use some WebGL application.

**Describe the bug**

I get the following error message in the Browser logs of the trace: "Failed to create WebGL context: Could not create a WebGL context."

* Switching back to `mcr.microsoft.com/playwright:v1.13.0-focal` all works fine within Docker.
* Running the same tests outside of Docker container works fine with v.1.16.0 on Windows.

**Comment by mxschmitt** [comment]
I tried to reproduce with the following which was working for me:

```shell
docker run -it --rm -it -v $(pwd):/work/ mcr.microsoft.com/playwright:v1.16.3-focal /bin/bash
mkdir foo
cd foo
npm init -y
npm install playwright@1.16.3
npx playwright install chrome
apt-get update && apt install nano # just some editor
nano test.js
```

```js
const playwright = require('playwright');

(async () => {
  const browser = await playwright.chromium.launch({
    channel: 'chrome',
  });
  const context = await browser.newContext();
  const page = await context.newPage();
  await page.goto('https://get.webgl.org/');
  await page.screenshot({ path: '/work/example.png' });
  await browser.close();
})();
```

**Comment by JoFrMueller** [comment]
Thanks @mxschmitt - if I run it like that it works. But can you please try this instead:

```bash
docker run -it --rm -it -v $(pwd):/work/ mcr.microsoft.com/playwright:v1.16.0-focal /bin/bash
mkdir foo
cd foo
npm init -y
npm install @playwright/test@1.16.0
npx playwright test
```

With these files in folder `foo`:

`playwright.config.ts`

```typescript
import { PlaywrightTestConfig } from '@playwright/test';

const config: PlaywrightTestConfig = {
  timeout: 60000,
  testMatch: '*.e2e-spec.ts',
  workers: 1,
  use: {
    trace: 'on',
  },
  expect: {
    toMatchSnapshot: {
      threshold: 0.2
    }
  },
  projects: [
    {
      name: 'Chrome Stable',
      use: {
        browserName: 'chromium',
        ignoreHTTPSErrors: true
      }
    }
  ]
};
export default config;
```

`no-no-it-is-broke.e2e-spec.ts`

```typescript
import { test } from '@playwright/test';

test.describe('No, no - it does not work', () => {

  test('matches expectation', async ({page}) => {
    await page.goto('https://get.webgl.org/');
    await page.waitForTimeout(1000);
    await page.screenshot({path: '/work/example.png'});
    await page.waitForTimeout(1000);
  });
});
```

**Comment by mxschmitt** [comment]
Yeah pretty sure that this was fixed in 1.16.2, see here: https://github.com/microsoft/playwright/releases/tag/v1.16.2

via 7927920c35290aa038f4052bc22c3ac1edf74993

**Comment by mxschmitt** [comment]
Closing since 1.16.3 should fix it.
